pkgs/top-level/python-packages.nix

```diff
@@ -5271,6 +5271,8 @@ in {
 
   prompt_toolkit = callPackage ../development/python-modules/prompt_toolkit { };
 
+  prompt_toolkit1 = callPackage ../development/python-modules/prompt_toolkit/1.nix { };
```

Comment:
I don't see this file being added.

Reply:
Because we already track it for the python2Packages set.

Reply:
this still has the pinning issues of people potentially using another package with the other major version, and either this package or the other package will break.

Really, python just needs to get it's packaging shit together.... but that ship sailed 20 years ago.

Reply:
Yeah, shit is on fire. :shrug: 

We don't need to merge this, I'm not sure I want to use it myself. I just thought I'd package it up openly instead of just for myself, but I haven't had time or place to try it out.

Reply:
If it's not critical to add now, I would probably just make an issue (if one doesn't already exists) to ask upstream to support the latest version.

(This is probably just me being bitter, but) If they didn't want to deal with constant dependency churn, upstream shouldn't have written an application in python :)

Reply:
Will do.

Reply:
that issue is from 3 years ago...

Reply:
The dependency got removed in 0.2.0! https://github.com/maubot/maubot/releases/tag/v0.2.0

Reply:
Feel free to pick this up if you're interested in maubot.